src/main/scala/scala/tools/refactoring/implementations/oimports/Region.scala

```diff
@@ -69,7 +69,7 @@ object Region {
 
   private def cutPrefixSuffix(imp: Global#Import, source: SourceFile): (String, List[String]) = {
     val printedImport = source.content.slice(imp.pos.start, imp.pos.end).mkString
-    val prefixPatternWithCommentInside = """import (((\/\*.*\*\/)*(\w|\d|_|-)+(\/\*.*\*\/)*)\.)+(\/\*.*\*\/)*""".r
+    val prefixPatternWithCommentInside = """import (((\/\*.*\*\/)*((\w|\d|_|-)+|(\`.*\`)+)(\/\*.*\*\/)*)\.)+(\/\*.*\*\/)*""".r
```

Comment:
The extra `\s+` is fine too... use whatever feels simpler for you, I just wanted to point out that `SourceWithMarker` might also be an option.

Moving the `Eye[..Of.type` problem to a separate PR is fine for me!